docs/lesson/contributing-to-obo-ontologies.md

```diff
@@ -8,8 +8,7 @@
 Participants will need to have access to the following resources and tools prior to the training:
 
 - **GitHub account** - register for a free GitHub account [here](https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home)
-- **Protege** - Install Protege 5.5, download it [here](https://protege.stanford.edu/)
-- **[Install ELK 0.5](../howto/installing-elk-in-protege.md)**
+- **Protege** - Install the latest version of Protege, download it [here](https://protege.stanford.edu/)
```

Comment:
I would recommend suggesting to download the latest version from the GitHub repository instead (https://github.com/protegeproject/protege-distribution/releases). The Protégé website at Stanford is not always updated quickly after a new release (right now, it still shows Protégé 5.6.1 as the latest version, even though 5.6.2 was released two months ago).